docs/howtoguides/attach_index.rst

```diff
@@ -0,0 +1,19 @@
+Attach a machine to a subscription
+**********************************
+
+* :ref:`How to attach <attach>` shows you how to perform the attach process
+* :ref:`How to simulate the attach operation <simulate-attach>` to check a
+  specific token to see what will happen if you attach to it (before
+  attaching anything), you can see:
```

Comment:
is this "you can see" supposed to be there? It is confusing me
```suggestion
* :ref:`How to simulate the attach operation <simulate-attach>` to check a
  specific token to see what will happen if you attach to it (before
  attaching anything)
```

Reply:
Also I think all bullets are supposed to end with a `.` if one of them does, right?

Reply:
Right on both counts! Thanks for spotting them